### DISCUSSION TITLE
Best way to post suggestions?

### DISCUSSION THREAD

**Discussion by obones** [toverainc/willow#141] [Q&A] [answered]
Hello,

I know this is a relatively new project and as such I believe you already have lots of ideas for improvements, better functionality and the like.
As I've been using it for the past week, I too have some ideas that would improve my own usage.
Before I gather up my ideas in a set of clean requests, I'd like to know what format you would prefer them in: Discussions, Issues, something else ?

Cheers

**Comment by kristiankielhofner** [comment, marked as answer]
I'd prefer we try to keep issues for immediately actionable items - bugs, features in progress, etc.

We can use discussions for back and forth on a given suggestion and if it becomes actionable we will create an issue from the discussion to track development progress.

**Reply by obones** [reply]
That's what I had in mind, I'll stick to it then

**Reply by obones** [reply]
Just one more thing: Am I right in assuming there is no point in starting discussions about items already in readme.md "The future" section?

**Reply by kristiankielhofner** [reply]
That's fine, please do!
